apps/editing-toolkit/.eslintrc.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
 
 		// FSE components render in a Gutenberg environment and should
 		// conform to those naming conventions instead of Calypso's.
-		'wpcalypso/jsx-classname-namespace': 0,
+		'wpcalypso/jsx-classname-namespace': 'off',
```

Comment:
My understanding is that '0' meant 'off' but I could be totally wrong here.

Reply:
Yeah, they seem to be equivalent: https://eslint.org/docs/latest/user-guide/configuring/rules